spec/features/users_spec.rb

```diff
@@ -0,0 +1,55 @@
+require "rails_helper"
+
+feature "user can navigate the app" do
+  scenario "user can create a new account" do
+    visit new_user_path
+
+    fill_in "user_name", with: "Mary"
+    fill_in "user_email", with: "Mary@example.com"
+    fill_in "user_password", with: "secret"
+    click_on I18n.t("users.new.submit")
+
+    expect(page).to have_text(I18n.t("users.show.welcome", name: "Mary"))
+    expect(page).
+      to have_text(I18n.t(
+        "flash.actions.create.notice",
+        resource_name: "User"
+      ))
+  end
+
+  scenario "user can view their show page" do
+    user = create(:user, name: "John")
```

Comment:
Since you're using `user.name` on line 24, setting the name here is not needed

Reply:
Done with sha: 4330bb5e0a56fc61dc659cf105600219ff954435